1-js/06-advanced-functions/12-arrow-functions/article.md

```diff
@@ -1,26 +1,26 @@
-# Arrow functions revisited
+# আবারো অ্যারো ফাংশন
 
-Let's revisit arrow functions.
+চলুন অ্যারো ফাংশন সম্পর্কে আরো কিছু জানা যাক।
 
-Arrow functions are not just a "shorthand" for writing small stuff. They have some very specific and useful features.
+শুধুমাত্র শর্টকাটে লিখার জন্য অ্যারো ফাংশন না। এর আরো কিছু বিশেষ ফিচার আছে।
 
-JavaScript is full of situations where we need to write a small function that's executed somewhere else.
+জাভাস্ক্রিপ্টে অনেক ক্ষেত্রে আমরা কোন একটি ফাংশনকে যথাযথ সম্ভব ছোট করে লিখতে চাই।
 
-For instance:
+যেমন:
 
-- `arr.forEach(func)` -- `func` is executed by `forEach` for every array item.
-- `setTimeout(func)` -- `func` is executed by the built-in scheduler.
-- ...there are more.
+- `arr.forEach(func)` -- `forEach` আর্গুমেন্ট হিসেবে একটি কলব্যাক ফাংশন নেয়।
+- `setTimeout(func)` -- `setTimeout` ও আর্গুমেন্ট হিসেবে একটি কলব্যাক ফাংশন নেয়।
+- ...এছাড়া আরো অনেক ক্ষেত্রে।
 
-It's in the very spirit of JavaScript to create a function and pass it somewhere.
+অনেক সময় আমাদের কোন ফাংশন লিখ তা অন্য কোন কন্টেক্সে ব্যবহার করতে হয়।
```

Comment:
Please change: `অনেক সময় আমাদের কোন ফাংশন লিখ তা অন্য কোন কন্টেক্সে ব্যবহার করতে হয়।`
to `অনেক সময় আমাদের কোন একটি ফাংশন লিখে তা অন্য আরেকটি ফাংশনের কন্টেক্সে ব্যবহার করতে হয়।`